README.md

```diff
@@ -34,6 +34,30 @@ To run PlaceCal locally you will need:
 
 With that said, here's what you need to get rolling.
 
+### Set up postgresql server (Via docker)
+
+**Note: Skip this step if you're using a system-installed docker**
+
+Creating a postgres docker image is simple:
```

Comment:
rule of thumb don't tell people what is simple, it can come across as a bit elitist if these are unfamiliar skills